.github/workflows/docker.yaml

```
@@ -3,7 +3,7 @@ name: Build and push Docker image
 on:
   push:
     branches:
-      - "**"
+      - "!dependabot/*"
```

This is required to be able to have it running on the other branches:

```suggestion
      - "**"
      - "!dependabot/*"
```